frontend/taipy-gui/src/components/Taipy/Metric.tsx

```diff
@@ -86,7 +87,7 @@ const Metric = (props: MetricProps) => {
                     prefix: extractPrefix(props.deltaFormat),
                     suffix: extractSuffix(props.deltaFormat),
                     valueformat: sprintfToD3Converter(props.deltaFormat)
-                },
+                } as Partial<Delta>,
```

Comment:
is this needed if `Data` is specified ?

Reply:
OK, that's kind of strange

Reply:
But it does appear in Plotly js docs, I guess they forgot to implement that when converting this into React component 🧐